src/components/table-component/table.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import Table from "./table";
+
+/* 
+ @author [Kurtney J.](https://github.com/xhundo)
+*/
+
+// Table Component StoryBook
+
+// Added prop control for avatar: visualization.
+export default {
+  title: "Components/Table",
+  component: Table,
+  argTypes: {
+    avatar: { control: "text" },
+    alt: { control: "text" },
+  },
+};
+
+export const Primary = (args) => (
+  <Table>
+    <Table.Header>Primary Accounts</Table.Header>
+    <Table.Cell>
+      <Table.Avatar src={args.avatar} alt={args.alt} />
```

Comment:
If possible, can we re-use the `Avatar` component here instead of duplicating it under the Table namespace? Julia created one in the `components/` directory (you may need to pull `main` into your branch if you don't have it already).

Reply:
Fixed